bin/validate_dn_format.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+import os
+import re
+
+regex = re.compile(r"^(/(?:DC|OU|C|ST|L|O)=[^/=]+)*(/CN=[^/=].+)$")
```

Comment:
I'm a little confused by the character class of `[^/=]`. Shouldn't it always be `/key=value`, i.e. `[^/]`?

```suggestion
regex_dn = re.compile(r"^(/(?:DC|OU|C|ST|L|O)=[^/=]+)*(/CN=[^/=].+)$")
```

It's always good to clarify what the regex is checking for.

Reply:
The `[^/=]` was to prevent cases where there would be a `=` in the value. E.g. `Some_key==value`. I assumed this would be an incorrect format as `=` was reserved for indicating key-value pairs.

Reply:
Ah gotcha, that LGTM then.